src/database/db_connection.js

```diff
@@ -0,0 +1,25 @@
+const { Pool } = require("pg");
+const url = require("url");
+require("env2")("./config.env");
+
+let DB_URL = process.env.DB_URL;
+
+if (!process.env.DB_URL)
+  throw new Error("Enviroment variable DB_URL must be set");
+  
+  if (process.env.NODE_ENV === "test") {
+    DB_URL = process.env.TEST_DB_URL;
+  }
+const params = url.parse(process.env.DB_URL);
+const [username, password] = params.auth.split(":");
+
+const options = {
+  host: params.hostname,
+  port: params.port,
+  database: params.pathname.split("/")[1],
+  user: username,
+  password,
+  ssl: true
```

Comment:
ssl would make a problem on localhost environment 
use this => `ssl :  options.host !== "localhost"`